addons/account/models/account_bank_statement.py

```diff
@@ -209,16 +253,23 @@ def unlink(self):
 
     @api.multi
     def open_cashbox_id(self):
+        self.ensure_one()
         context = dict(self.env.context or {})
-        if context.get('cashbox_id'):
-            context['active_id'] = self.id
+        if context.get('balance'):
+            context['statement_id'] = self.id
+            if context['balance'] == 'start':
+                cashbox_id = self.cashbox_start_id.id
+            elif context['balance'] == 'close':
+                cashbox_id = self.cashbox_end_id.id
+            else:
+                cashbox_id = False
             return {
                 'name': _('Cash Control'),
                 'view_mode': 'form',
                 'res_model': 'account.bank.statement.cashbox',
-                'view_id': self.env.ref('account.view_account_bnk_stmt_cashbox').id,
+                'view_id': self.env.ref('account.view_account_bnk_stmt_cashbox_close_modal').id,
```

Comment:
this view: account.view_account_bnk_stmt_cashbox_close_modal' doesn't exists, you have create point_of_sale.view_account_bnk_stmt_cashbox_close_modal in point_of_sale and you cannot and don't want to call it in account. you have to let the 'account.view_account_bnk_stmt_cashbox' in accounting and call your new view in POS module.

Moreover, you are not calling this method from POS, so no need to modify the view.